cmake/CMakeLists.txt

```diff
@@ -429,7 +444,17 @@ endforeach()
 
 set(IFCPARSE_FILES ${IFCPARSE_CPP_FILES} ${IFCPARSE_H_FILES})
 
-ADD_LIBRARY(IfcParse STATIC ${IFCPARSE_FILES})
+if (BUILD_SHARED_LIBS)
+    add_library(IfcParse SHARED ${IFCPARSE_FILES})
+    set(IFCBINS "${IFCBINS};IfcParse")
```

Comment:
I don't fully understand: `IfcParse` is added to `IFCBINS` now on non-MSVC platforms. With the `INSTALL` statement below this means that `IfcParse.lib` would go to `/usr/bin`, whereas I think the convention is to put it in `/usr/lib`? Somehow this also triggers a CMake error in Linux: `install TARGETS given no LIBRARY DESTINATION for shared library target "IfcParse"`, potentially because of the empty `${IFCLIBS}` at the `INSTALL` command, not sure.